docs/en/llm/api_server_reasoning.md

```diff
@@ -0,0 +1,109 @@
+# Reasoning Outputs
+
+For models that support reasoning capabilities, such as [DeepSeek R1](https://huggingface.co/deepseek-ai/DeepSeek-R1), LMDeploy supports parsing the reasoning results in the service and separately records the reasoning content using `reasoning_content`.
+
+## Examples
+
+### DeepSeek R1
+
+We can start the DeepSeek R1 model's api_server service just like launching other models. The difference is that we need to specify `--enable-reasoning` and `--reasoning-parser` parameters.
+In the `--reasoning-parser` parameter, we need to specify the specific parser to use.
+
+```
+lmdeploy serve api_server deepseek-ai/DeepSeek-R1-Distill-Qwen-1.5B \
```

Comment:
should it be `deepseek-r1` instead of `deepseek_r1`? 

```
ValueError: The reasoning parser deepseek_r1 is not in the parser list: dict_keys(['deepseek-r1', 'qwen-qwq'])
```